pages/message-board.html

```diff
@@ -35,13 +34,50 @@
     <main>
       <h1>The Regis Altare Birthday Project 2023</h1>
 
+      <a href="#message-board" class="visually-hidden">Skip to content</a>
+
+      <!-- Inverted border radius for clip-path -->
+      <svg width="0" height="0" aria-hidden="true">
+          <clipPath id="navClip" clipPathUnits="objectBoundingBox">
+              <path transform="scale(0.004388274530454626, 0.019230769230769232)" d="M173.88,0H54C38,0,45,21,0,52H227.88C182.88,21,189.88,0,173.88,0Z"/>
+          </clipPath>
+      </svg>
+
+      <!-- Burger menu for responsive layout -->
+      <button id="nav-button" aria-label="Toggle site navigation">
+          <span class="visually-hidden">Menu</span>
+          <img class="icon icon-menu" aria-hidden="true" src="../icons/Icons_Hamburger.svg"/>
+      </button>
+
+      <!-- Navigation -->
       <nav>
-        <a href="../pages/collection.html">Collection</a>
-        <a href="../pages/gacha.html">Gacha</a>
-        <a class="active" href="../pages/message-board.html">Birthday Message Board</a>
+          <a id="nav-home" href="../index.html">
+            <img class="icon icon-logo" aria-hidden="true" src="../icons/Icons_Open-Packs.svg"/>
+            <span>Home</span>
+          </a>
+          <a id="nav-gacha" href="../pages/gacha.html">
+              <img class="icon icon-gacha" aria-hidden="true" src="../icons/Icons_Open-Packs.svg"/>
+              Gacha</a>
```

Comment:
I didn't read the new nav css too closely so can't pinpoint the cause, but I noticed while tabbing around that the keyboard focus highlight is dark blue/ hard to see on the nav items specifically, while other buttons have the default black/ white stroked highlight. Would this be a quick fix?

Reply:
Oh jeez, good catch! Looks like applying `clip-path` on the `a` tags was causing the keyboard focus outlines to not render. I'll move the `clip-path` styling to a ::before psuedo element.